packages/driver/src/cross-origin/patches/cookies.js

```diff
@@ -15,7 +13,7 @@
 // - On an interval, get the browser's cookies for the given domain, so that
 //   updates to the cookie jar (via http requests, cy.setCookie, etc) are
 //   reflected in the document.cookie value.
-export const patchDocumentCookie = (Cypress) => {
+export const patchDocumentCookie = (Cypress, window) => {
```

Comment:
these patches are pretty straight forward and a good way to keep the injected code small 👍